db/connect.go

```diff
@@ -47,15 +47,6 @@ func ConnectDB(p params.Database) (Database, error) {
 	conn.SetMaxIdleConns(p.MaxIdleConnections)
 	conn.SetConnMaxLifetime(p.ConnectionMaxLifetime)
 
-	if !p.DisableAutoMigrate {
```

Comment:
Missing the code that removes the member from the struct and the loading from the configuration